controller.go

```diff
@@ -115,6 +115,7 @@ func (c *Controller) Init(ctx *context.Context, controllerName, actionName strin
 	c.EnableXSRF = true
 	c.Data = ctx.Input.Data()
 	c.methodMapping = make(map[string]func())
+	c.CruSession = ctx.Input.CruSession
```

Comment:
If assign c.CurSession here, no need assign again in Controller.StartSession(), no matter BConfig.WebConfig.Session.SessionOn